### ISSUE TITLE
Please give example how to run

### ISSUE THREAD

**Issue by bertdouglas** [swedgwood/solitaire-web#2]
I cloned the project from github.  Then did "cargo build".  It worked.  Now I am stuck.  What do I do next to see it run?  Thanks. 

I understand that I need a local webserver.  There are so many.  I tried installing trunk, because that seems be used a lot with yew.  It installed ok.  But I can't figure out how to use it.  The index.html file does not load any webassembly.  It only has css.

**Comment by swedgwood** [comment]
Hey! Haven't touched this in a good few years, but I *think* that if you want to run it locally with `trunk`, you can just do `trunk serve` in the project directory (for me this serves it at `http://127.0.0.1:8080`)

And if you want to build it statically, you do `trunk build --release --public-url <path you're hosting this at>`. So I've got it hosted at `https://projects.wedgwood.dev/solitaire/`, so for me its `trunk build --release --public-url /solitaire`, which puts the built website into `dist`. I haven't tested it but I imagine if you want to access it directly (via `file://`) then you'd probably need to give the full path the the `dist` folder.

For more details it's probably good to take a look at `trunk serve --help` or `trunk build --help` for whichever one you wanted to use.

**Comment by bertdouglas** [comment]
Thanks much.  Wow.  I am most pleasantly surprised to get a reply.
I found the "workflow" file and copied a line out of there and got it to
work.  Very nice.
I figured out how to move cards among the tableau with space key.  Also how
to move to foundation with enter key.
But I was not able to figure out how to get a card from the stock.

Any way, it is amazing that rust code, that is years old, compiled with no
errors.

I want to make an automatic solver for solitaire that is guaranteed to find
a solution if one exists.  Perhaps the search tree will be too big, but I
think it branches less than chess or go.

Thanks for putting the code up.  Will study it more.
